CalibTracker/SiStripLorentzAngle/plugins/MeasureLA.cc

```diff
@@ -1,10 +1,17 @@
 #include "CalibTracker/SiStripLorentzAngle/plugins/MeasureLA.h"
 #include "CalibTracker/SiStripLorentzAngle/interface/LA_Filler_Fitter.h"
 #include "CalibTracker/SiStripCommon/interface/SiStripDetInfoFileReader.h"
+#include "DataFormats/TrackerCommon/interface/LegacyTrackerTopology.h"
+
 #include <boost/lexical_cast.hpp>
 #include <TChain.h>
 #include <TFile.h>
 
+namespace {
```

Comment:
@alesaggio is this still needed after having introduced the LegacyTopology stuff?